packages/ui/src/assets/icons/index.ts

```diff
@@ -1,5 +1,7 @@
+export { default as Add } from "./Add.svg?react"
```

Comment:
For svg we use lucide library. if it's not a specific case like chevron, user, add, etc. use them from that library. It can be a little be different from figma